src/net/fe/overworldStage/Zone.java

```diff
@@ -234,8 +221,7 @@ public Fog(Set<Node> zone) {
 		HEAL_DARK(0xC044FF44),
```

Comment:
That enum is unrelated to Zone now. It should probably be renamed and moved in RangeIndicator.

Reply:
Github is hard, this was meant to be on the enum 
net.fe.overworldStage.Zone.ZoneType